scripts/local_environment.sh

```diff
@@ -18,21 +18,33 @@ function usage() {
     ----------
     --deploy        - Perform creation of test container
     --destroy       - Perform destroy of test container
-    --get-runtime   - Get available container runtime
+    --runtime       - Sets runtime to be used.
+                      Supported args: docker/podman
+                      By default - "docker"
     --help|-h       - Print help
 EOF
 }
 
-function check_container_engine() {
-    if command -v podman &> /dev/null; then
-	RUNTIME=podman
-    elif command -v docker &> /dev/null; then
-        RUNTIME=docker
+function sets_container_engine() {
+    if [[ "$RUNTIME" == "docker" ]]; then
+        if [[ -x "$(command -v docker)" && -S /var/run/docker.sock ]]; then
+            RUNTIME="docker"
+        else
+            echo "Docker engine is not available"
+            exit 1
+        fi
```

Comment:
`RUNTIME` is already set, so the `if` branch isn’t needed

```suggestion
        if ! [[ -x "$(command -v docker)" && -S /var/run/docker.sock ]]; then
            echo "Docker engine is not available"
            exit 1
        fi
```

This applies to the `podman` test too.

Reply:
Fixed.